tools/release-operator/src/registry.rs

```diff
@@ -53,6 +54,17 @@ impl Registry {
     }
 }
 
+#[derive(Debug)]
+struct CrateNotFoundError;
+
+impl std::fmt::Display for CrateNotFoundError {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        write!(f, "crate was not found on crates.io")
+    }
+}
+
+impl std::error::Error for CrateNotFoundError {}
+
```

Comment:
The same thing could achieve in a more compact way with `thiserror`, but this is fine too.

Reply:
I did it like this, because I did not want to pull in another dependency for a single error. 

However, I will check if the crate is already part of the dependency tree because of something else. If so, we can just as well use it.

Reply:
It's already used in a bunch of places. At least `fj-kernel`, and I think `fj-viewer` too.

In any case, not a big deal. I just wanted to mention the possibility.

Reply:
Pulled it in 😇